package.json

```diff
@@ -99,7 +99,7 @@
     ]
   },
   "dependencies": {
-    "@rsksmart/rif-relay-client": "2.0.0-beta.6",
+    "@rsksmart/rif-relay-client": "github:rsksmart/rif-relay-client#fix/verify_with_relay_hub",
```

Comment:
It needs to be updated after merging the related PR

Reply:
updated